napari/_qt/_qapp_model/_tests/test_file_menu.py

```diff
@@ -284,7 +285,9 @@ def test_open_with_plugin(
     stack,
 ):
     viewer = make_napari_viewer()
```

Comment:
Are we able to delete the `get_open_with_plugin_action` function?

Reply:
Yep!